Services/Interfaces/web_interface/advanced_controllers/tentacles.py

```diff
@@ -18,15 +18,34 @@
 
 import tentacles.Services.Interfaces.web_interface.advanced_controllers as advanced_controllers
 import tentacles.Services.Interfaces.web_interface.login as login
+from tentacles.Services.Interfaces.octo_ui2.models.octo_ui2 import import_cross_origin_if_enabled, dev_mode_is_on
```

Comment:
I think we should not import octo_ui2 here. We should load it dynamically as a web interface plugin